src/components/ProductMilestoneCreateEditPage/ProductMilestoneCreateEditPage.tsx

```diff
@@ -224,21 +217,33 @@ export const ProductMilestoneCreateEditPage = ({ isEditPage = false }: IProductM
             </FormHelperText>
           }
         >
-          <InputGroup>
-            <InputGroupText>
+          <TextInputLoading
+            isRequired
+            type="text"
+            id={productMilestoneEntityAttributes.version.id}
+            name={productMilestoneEntityAttributes.version.id}
+            autoComplete="off"
+            {...register<string>(productMilestoneEntityAttributes.version.id, {
+              ...fieldConfigs.version,
+              validators: [
```

Comment:
There is none, but you can easily create one locally.

Reply:
Please can you put the one on demo page so that it will be available as reference example for everyone? I don't want to create it myself locally. I want everyone working on the UI can see some working example + we use demo page also for manual testing purposes.